### ISSUE TITLE
Question about model output when evaluating the model

### ISSUE THREAD

**Issue by govindamagrawal** [david8862/keras-YOLOv3-model-set#14]
Hi David,
I am training tiny-yolo on my own dataset but not getting good accuracy. I am trying to find mAP per class but got confused regarding the model output. In the 'eval.py', for tiny-yolo:
if len(anchors) == 6:
        output_tensor_names = ['conv2d_1/Conv2D', 'conv2d_3/Conv2D']

So, it has 2 output tensors. But during the model summary the tiny-yolo, the last few lines in model.summary are:
.............................................
leaky_re_lu_8 (LeakyReLU)       (None, 13, 13, 512)  0           batch_normalization_8[0][0]      
________________________________________________________________________________
leaky_re_lu_10 (LeakyReLU)      (None, 26, 26, 256)  0           batch_normalization_10[0][0]     
________________________________________________________________________________
prediction_13 (Conv2D)          (None, 13, 13, 255)  130815      leaky_re_lu_8[0][0]              
________________________________________________________________________________
prediction_26 (Conv2D)          (None, 26, 26, 255)  65535       leaky_re_lu_10[0][0]             

So, accordingly, the output tensor should be 'prediction_26'. Can you please explain, how you got 'conv2d_1/Conv2D', 'conv2d_3/Conv2D' as output tensor?

**Comment by david8862** [comment]
> Hi David,
> I am training tiny-yolo on my own dataset but not getting good accuracy. I am trying to find mAP per class but got confused regarding the model output. In the 'eval.py', for tiny-yolo:
> if len(anchors) == 6:
> output_tensor_names = ['conv2d_1/Conv2D', 'conv2d_3/Conv2D']
> 
> So, it has 2 output tensors. But during the model summary the tiny-yolo, the last few lines in model.summary are:
> .............................................
> leaky_re_lu_8 (LeakyReLU) (None, 13, 13, 512) 0 batch_normalization_8[0][0]
> 
> leaky_re_lu_10 (LeakyReLU) (None, 26, 26, 256) 0 batch_normalization_10[0][0]
> 
> prediction_13 (Conv2D) (None, 13, 13, 255) 130815 leaky_re_lu_8[0][0]
> 
> prediction_26 (Conv2D) (None, 26, 26, 255) 65535 leaky_re_lu_10[0][0]
> 
> So, accordingly, the output tensor should be 'prediction_26'. Can you please explain, how you got 'conv2d_1/Conv2D', 'conv2d_3/Conv2D' as output tensor?

Hi @govindamagrawal , what's your model type and training config cmd? The tensor name you mentioned is hardcoded for some model format (.pb, .mnn) which need to specify input/output tensor for inference, so it may need to be adjust if it doesn't match your real model output.
